blueprints/addon/files/README.md

```diff
@@ -7,7 +7,7 @@
 Compatibility
 ------------------------------------------------------------------------------
 
-* Ember.js v3.16 or above
+* Ember.js v3.20 or above
 * Ember CLI v2.13 or above
```

Comment:
Mind sending another PR updating this to current versions too?

Reply:
Sure, which version should this be?
Was also wondering if we could automatically fill this in when generating an addon, one less thing to update manually when we have a new LTS version.

Reply:
Should be in sync with ember version.